src/pages/hegel/reference/nothing/development.mdx

```diff
@@ -52,16 +53,61 @@ Hegel goes to add a link between the empty thinking (or empty intuiting) that fo
+
+### Houlgate
+
+Stephen Houlgate offers a detailed examination of the development of `nothing`. In contrast to Parmenides, whose idea of `nothing` is that of a contrast to `being`[^2], Houlgate points out that for Hegel, `nothing` is not to be regarded as a negative or absence of something but "a negative from which being is altogether absent ... the sheer and utter _nothing_, in which there is no trace of being whatsoever" (Houlgate 2022, 132).
+
+[^2]: "Parmenides conceives being from the outset in opposition to nothing: being _is_, whereas nothing is _not_. Furthermore, this opposition is absolute: being does not emerge from or pass into nothing, but remains always and only being. Being _is_, and it never proves to be anything other than being: it is changeless, 'uncreated and imperishable'" (Houlgate 2022, 138).
+
+Having established the absolute nothingness of `nothing`, Houlgate next points out that, by virtue of its purity as absolute nothingness, "nothing has an _immediacy_ to it: it is simply and immediately nothing, and nothing else besides" (Houlgate 2022, 132). Thus, this precise _immediacy_ proves `nothing` to be:
+
+> utterly indeterminate _being_: for simple immediacy, at this stage in speculative logic, is all that being is understood to be (Houlgate 2022, 132).
+
+This sense of immediacy is further stressed through the self-equality of `nothing`. This simple quality with itself at once means that `nothing` is _immediately itself_. Houlgate additionally refers to §88 of Hegel's _Encyclopaedia Logic_, where, Hegel writes: "And similarly, but conversely, nothing, as this immediate [term] that is equal to itself, is the same as being" (Hegel 1991, 141/§88). And so Houlgate concludes that, "just as pure being, by virtue of its indeterminacy, vanishes into nothing, so pure nothing, by virtue of its immediacy, vanishes into being" (Houlgate 2022, 143).
+
+Houlgate additionally notes that while Hegel may seem to suggest that pure nothing turns to being because, as Hegel's text shows, "nothing _is_ (exists) in our intuiting or thinking", he reminds us that speculative logic does not provide an account of what happens to being and nothing when they are thought _by us_ human beings. The unfolding of these categories, their "fate" as it were, certainly requires human thought, but it is not human thought that determines what these categories are.
+
+> The development Hegel traces ... is not a product of our thinking; rather, it is made necessary, logically, by being and nothing themselves and is simply made explicit and articulated by thought. In speculative logic, our thought does not determinate the development that the categories of being and [nothing][^3] undergo, but it is itself determined by the immanent logical development of those categories (Houlgate 2022, 143).
+
+[^3]: The passage reads "being and thought" but thought is not a category, so this must be a textual error (Niklas).
```

Comment:
I think he does intend to write "thought" because he's talking about speculative logic as such, and speculative logic as such deals with the categories of being and thought.

Reply:
But thought is not a category! Kindly point to the place in the _Logic_ where thought is explicated as a category. 😀

Reply:
Hahha, but as you know, Houlgate refers to the subject matter of the Logic as the determinations of thought and being, and I think that's what's going on here.

Reply:
Right, but I still think that's a different context. Categories here should refer to the explicitly annoted categories in the text. We'll have to take it up with the source itself then!